lib/db_utils.go

```diff
@@ -174,6 +174,12 @@ type DBPrefixes struct {
 	PrefixLikerPubKeyToLikedPostHash []byte `prefix_id:"[30]" is_state:"true"`
 	PrefixLikedPostHashToLikerPubKey []byte `prefix_id:"[31]" is_state:"true"`
 
+	// Prefixes for reactions:
+	// <prefix, user pub key [33]byte, reacted post hash [32]byte> -> <>
+	// <prefix, post hash [32]byte, user pub key [33]byte> -> <>
+	PrefixReactorPubKeyToPostHash []byte `prefix_id:"[63]" is_state:"true"`
+	PrefixPostHashToReactorPubKey []byte `prefix_id:"[64]" is_state:"true"`
```

Comment:
We need to include react emoji in these indices. I think the correct indices would be
1. PrefixReactorPubKeyToPostHashReactEmojiL <prefix, user pub key, reacted post hash, react emoji>
2. PrefixPostHashReactEmojiToReactorPubKey <prefix, reacted post hash, react emoji, user pub key>

this allows us to get all the different reactions a user gave to a single post easily (index 1) as well as get a list of all users who gave a "happy" react to a given post (index 2)

Reply:
Additionally, I would scoot these to the bottom of the list right before "NEXT_TAG"